.github/workflows/web-bff-ci.yml

```
@@ -0,0 +1,38 @@
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'web-bff/**' # Trigger only for changes in the web-bff directory
+  pull_request:
+    branches:
+      - main
+      - '**' # Trigger for all PR branches
```

```suggestion
    branches:
```
Removing the options will trigger for all branches I think, you can try once.

---

sorry can't get you..

---

I agree - removing the entries for the `branches` node will mean that it will trigger for all branches

---

done